pkg/types/utils.go

```diff
@@ -32,3 +39,19 @@ func MachineNetworksToCIDRs(nets []MachineNetworkEntry) []configv1.CIDR {
 
 	return res
 }
+
+// GetClusterProfileName utility method to retrieve the cluster profile setting.  This is used
+//
+//	when dealing with openshift api to get FeatureSets.
+func GetClusterProfileName() configv1.ClusterProfileName {
+	// Get cluster profile for new FeatureGate access.  Blank is no longer an option, so default to
+	// SelfManaged.
+	clusterProfile := configv1.SelfManaged
+	if cp := os.Getenv("OPENSHIFT_INSTALL_EXPERIMENTAL_CLUSTER_PROFILE"); cp != "" {
```

Comment:
nit: could we change this to `OPENSHIFT_INSTALL_CLUSTER_PROFILE`? I suspect the installer will need to add an option for this in the install config, so thank you for making this configurable until we can do that!

Reply:
Yes.  I saw there was an existing profile. I saw it does something similarly.  Do I just update it here or should we look into updating other spot too?  https://github.com/openshift/installer/blob/024386d953e6615e0d324b3867773c20ce35ea77/pkg/asset/ignition/bootstrap/common.go#L311

Reply:
Ah! I didn't realize this was using an existing env var. In that case, let's just keep it as is.